csrc/multidevice/executor.cpp

```diff
@@ -277,4 +277,18 @@ std::ostream& MultiDeviceExecutor::print() {
   return debug();
 }
 
+std::vector<FusionExecutorCache*> MultiDeviceExecutor::
+    getFusionExecutorCaches() {
+  NVF_CHECK(
+      params_.use_fusion_executor_cache,
+      "MultideviceExecutor must be configured to use FusionExecutorCache");
+  std::vector<FusionExecutorCache*> fecs;
+  for (SegmentedGroup* group : workspace_.group_run_order) {
+    if (!is_resharding_.at(group)) {
```

Comment:
What do you think about
```suggestion
    if (fec_.count(group) > 0) {
```